develop/devguide/Connector/ConnectorHelp/Connector_help_pages.md

```diff
@@ -51,33 +60,61 @@ Use DocFX Flavored Markdown (DFM). See [Markdown syntax](xref:CTB_Markdown_Synta
 
 ### Title
 
-The title of the page should be the name of the connector in the [DataMiner Catalog](https://catalog.dataminer.services/).
+The title of both pages should be the name of the connector in the [DataMiner Catalog](https://catalog.dataminer.services/).
+
+### Marketing page
```

Comment:
This more or less in line with https://docs.dataminer.services/user-guide/Cloud_Platform/Catalog/Best_Practices_When_Documenting_Catalog_Items.html.
Can we not simply make a reference to these guidelines?